examples/test/remote/index.rsh

```diff
@@ -0,0 +1,62 @@
+'reach 0.1';
+
+//Request, allowace and approve functions to match index.sol
+export const main =
+
+Reach.App(
+  {},
+  [Participant('Child', { request: Fun([UInt], UInt) } ),
+  Participant('Parent', { allowance: Fun([], UInt),
+    getSolAddress: Fun([], Address),
+    getSolContract: Fun([], Tuple(UInt, Address)),
+    approve: Fun([UInt, UInt], Bool),
+    ...hasConsoleLogger,
+  } )],
+    (Child, Parent) => {
+      Child.publish();
+      
+      commit();
+      
+      
+      Parent.only(() => {
+        //remote object
+        const solAddr = declassify(interact.getSolAddress());
+        const signSolContract = declassify(interact.getSolContract());
+        const allowance = declassify(interact.allowance()); });
+        Parent.publish(allowance, signSolContract, solAddr)
+        .pay(allowance);
+        const sign = remote(solAddr, {});
+        Parent.interact.log(allowance);
+        Parent.interact.log(solAddr);
+    
+
+
+        var [ bal, oks, nos ] = [ allowance, 0, 0 ];
```

Comment:
This is residue from the example you copied, but you just need a really simple Reach program with two steps.